script/deploy/zksync/utils/ScriptBase.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { Script, console } from "forge-std/Script.sol";
+import { DSTest } from "ds-test/test.sol";
+import { LibAsset } from "lifi/Libraries/LibAsset.sol";
+import { stdJson } from "forge-std/Script.sol";
+
+contract ScriptBase is Script, DSTest {
+    using stdJson for string;
+
+    error NotAContract(string key);
```

Comment:
the error could be removed as it's unused.

Reply:
same in `script/deploy/facets/utils` but I am not sure If it should be part of this PR

Reply:
should be a different PR imo

Reply:
then I am good